feature-detects/battery.js

```diff
@@ -16,5 +16,5 @@
 Detect support for the Battery API, for accessing information about the system's battery charge level.
 */
 define(['Modernizr', 'prefixed'], function(Modernizr, prefixed) {
-  Modernizr.addTest('batteryapi', !!prefixed('battery', navigator), {aliases: ['battery-api']});
+  Modernizr.addTest('batteryapi', !!prefixed('battery', navigator) || !!prefixed('getbattery', navigator), {aliases: ['battery-api']});
```

Comment:
sorry, didn't notice before. But this should be `getBattery`, not `getbattery`

Reply:
Oh lol, I should have known. :P Thanks